README.md

```diff
@@ -1,2 +1,254 @@
 # smartclide-context
 SmartCLIDE Context Handling Component
+
+## Preconditions to build and run Context Handling
+
+To build and run Context Handling, the following software is required:#
+
+- Java (at least version 11)
+- Apache Maven (min V3.6)
+- Docker [Desktop] (for running Context Handling)
```

Comment:
Is docker really needed to run context-handling?

Reply:
No it is not. Only required for deploing it on our Cluster